bin/ncp/NETWORKING/dnsmasq.sh

```diff
@@ -73,9 +73,9 @@ EOF
   update-rc.d dnsmasq defaults
   update-rc.d dnsmasq enable
   service dnsmasq restart
-  cd /var/www/nextcloud
-  sudo -u www-data php occ config:system:set trusted_domains 2 --value=$DOMAIN
-  sudo -u www-data php occ config:system:set overwrite.cli.url --value=https://"$DOMAIN"/
+  ncc config:system:set trusted_domains "${TRUSTED_DOMAINS[dnsmasq]}" --value="$DOMAIN"
+  #TODO: Is this really desirable?
+  set-nc-domain "$DOMAIN" --no-trusted-domain
```

Comment:
Do we really want to overwrite the main domain for nextcloud with the dnsmasq domain? As I understand it, this is the domain of the dns server and there are use cases where the two might differ (e.g. if you only want to use the dns server in the local network and aren't using a global IP/domain)

Reply:
yes, we want this for users to be able to use the same domain internally and externally when they don't have hairpin NAT